src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,27 @@
+package core.basesyntax;
+
+import core.basesyntax.model.Circle;
+import core.basesyntax.model.Figure;
+import core.basesyntax.model.Rectangle;
+import core.basesyntax.model.Square;
+import core.basesyntax.supplier.FigureSupplier;
+
+public class Main {
+    private static final FigureSupplier FIGURE_SUPPLIER = new FigureSupplier();
+    private static final int NUMBER_OF_FIGURES = 6;
+
+    public static void main(String[] args) {
+
+        Figure[] figures = new Figure[NUMBER_OF_FIGURES];
+        figures[0] = FIGURE_SUPPLIER.getRandomFigure();
+        figures[1] = FIGURE_SUPPLIER.getRandomFigure();
+        figures[2] = FIGURE_SUPPLIER.getRandomFigure();
+        figures[3] = new Circle("WHITE", 6);
+        figures[4] = new Square("RED", 4);
+        figures[5] = new Rectangle("GREEN", 5, 5);
+
+        for (Figure figure: figures) {
+            figure.draw();
+        }
```

Comment:
<img width="782" alt="Screen Shot 2022-11-03 at 9 03 01 PM" src="https://user-images.githubusercontent.com/90692308/199811551-707e0106-9573-4b3a-8ad1-803fba7a33d4.png">
Nastya has written what it should look like